test/dynamo/test_misc.py

```diff
@@ -2967,6 +2967,19 @@ def fn(x):
         res = opt_fn(x)
         self.assertTrue(same(ref, res))
 
+    @unittest.skipIf(not torch.backends.cudnn.is_available(), "requires cudnn")
+    def test_torch_cudnn_is_acceptable(self):
+        def fn(x):
+            if torch.backends.cudnn.is_acceptable(tensor=x):
+                return x.relu()
```

Comment:
```suggestion
                return x+1
```
will make this test lest flakey.  There is a small chance `x == x.relu()` if x is all positive.